src/pipe_log.rs

```diff
@@ -219,7 +219,7 @@ impl LogManager {
 }
 
 #[derive(Clone)]
-pub struct PipeLog {
+pub struct PipeLogImpl {
```

Comment:
Prefer using a more specific name like `FilePipeLog` as there could be `MemPipleLog`

Reply:
> Prefer using a more specific name like `FilePipeLog` as there could be `MemPipleLog`

Renamed!

Reply:
@Renkai I guess you made a mistake. Should be `FilePipeLog` instead of `MemPipeLog`. The latter can be added later for benchmarks.